tfx/components/example_validator/component.py

```diff
@@ -68,6 +69,8 @@ class ExampleValidator(base_component.BaseComponent):
   def __init__(self,
                statistics: types.Channel = None,
                schema: types.Channel = None,
+               # pylint: disable=dangerous-default-value
+               exclude_splits: Optional[List[Text]] = ['train'],
```

Comment:
if we don't allow None, we can set it to, then it's not a dangerous default?

exclude_splits:List[Text] = ['train'] (user can set it to [] if they don't want to exclude)

same elsewhere

Reply:
then should i change the parameters in the spec so optional is set to False? I will update design doc as well?

Reply:
I kept the pylint issue for now and let's see what other reviewers prefer.

Reply:
sgtm, 
(discussed offline, the other option is 
use None as default (will be changed to ['train']) and use [] as no exclude splits, in this option, none and empty means different things.)

Reply:
Don't use list for default value because it's **mutable**

https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

Use None here and make it clear that this would default to ['train'] if not set

Reply:
Thanks. I understand. I will make the change now.